LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2019 FiF0o <jonathan.lazarini@wipro.com>
```

Comment:
Probably should but "Buildit" as the copyright holder (our `gravity-ui-web` repo does this: https://github.com/buildit/gravity-ui-web/blob/develop/LICENSE).

Strictly speaking, this probably needs to be "Wipro Limited" or something like that. But, until we get around to checking with legal or whoever, let's at least make our licenses consistent. If/when we amend them later on, we can do all at the same time.

Reply:
Resolving as this is part of the other branch now since we split it into 2 @james-nash